MinimOsd_Extra/protocols/NMEA.h

```diff
@@ -249,7 +233,7 @@ store to msg.nmea.lat
             break;
 
         case 6:
-            if (frame == FRAME_GGA) {  msg.nmea.fix = (string[0] - '0');         }
+            if (frame == FRAME_GGA) {  msg.nmea.fix = (string[0] - '0');}
             if (frame == FRAME_RMC && string[0] == 'W') msg.nmea.lon = -msg.nmea.lon;
```

Comment:
And why did you interfere with the gaps? Amendments to the style change are not accepted.